app/Exports/EnrollmentsExport.php

```diff
@@ -8,9 +8,12 @@
 use Maatwebsite\Excel\Concerns\WithMapping;
 use Maatwebsite\Excel\Concerns\WithHeadings;
 use Maatwebsite\Excel\Concerns\FromCollection;
+use Maatwebsite\Excel\Concerns\Exportable;
 
 class EnrollmentsExport implements FromCollection, WithMapping, WithHeadings
 {
+    use Exportable;
```

Comment:
Is this mandatory?

Reply:
In _Http/Controllers/Admin/EnrollmentController.php_, the export is done by assuming that the EnrollmentsExport has this trait, as in:

`(new EnrollmentsExport())->download`

To maintain that structure, this change is mandatory. However, we could not include the Exportable trait if we instead used the _Excel::download_ facade, like this (in the controller):

`Excel::download(new EnrollmentsExport, 'enrollments.csv', ...)`

For reference: [Exporting collections](https://docs.laravel-excel.com/3.0/exports/collection.html)